Ops/Expr/Graph.cs

```diff
@@ -24,6 +24,14 @@ public override int Visit (NIdentifier ident)
       return id; 
    }
 
+   public override int Visit (NFnCall function) {
+      List<int> args = new (); 
+      function.Params.ToList ().ForEach (a => args.Add (a.Accept (this)));

```

Comment:
A neater way to do this is:
`var args = function.Params.Select (a => a.Accept(this)).ToList ()`

Reply:
Changed